tmt/convert.py

```diff
@@ -489,8 +491,14 @@ def read_nitrate_case(testcase):
         echo(style('extra-summary: ', fg='green') + data['extra-summary'])
     # Contact
     if testcase.tester:
-        data['contact'] = '{} <{}>'.format(
-            testcase.tester.name, testcase.tester.email)
+        if testcase.tester.name is not None:
+            data['contact'] = '{} <{}>'.format(
+                testcase.tester.name, testcase.tester.email)
+        else:
+            if not re.search(testcase.tester.email, str(makefile_contact[0])):
+                data['contact'] = testcase.tester.email
+            else:
+                data['contact'] = makefile_contact[0]
```

Comment:
This branch will fail if there is no `Owner` (from Makefile) and no testcase.tester (the nitrate one), because then the `global makefile_contact` is not initialized.